packages/client/src/components/note.vue

```diff
@@ -452,6 +456,21 @@ function readPromo() {
 				}
 
 				> .content {
+					&.isLong {
+						> .showLess {
+							width: 100%;
+							margin-top: 1em;
```

Comment:
I think it would be even better if you can make the button for collapsing sticky. I.e. it does not go out of view as long as you can see the note. This would be helpful because you can immediately collapse something again without having to scroll all the way to the bottom.

I think something like this should work:
```suggestion
							margin-top: 1em;
							position: sticky;
							bottom: 1em;
```
But then the button gets hard to see when its over the content, so maybe also increase the shadow.

Reply:
With that sticky addition, I love it even more 😆